Python/Product/Analysis/Analyzer/DDG.cs

```diff
@@ -304,6 +304,20 @@ internal class DDG : PythonWalker {
         public override bool Walk(FromImportStatement node) {
             var modName = node.Root.MakeString();
 
+            var candidates = ModuleResolver.ResolvePotentialModuleNames(_unit.ProjectEntry, modName, node.ForceAbsolute).ToArray();
+            if (candidates.Length == 1 && string.IsNullOrEmpty(candidates[0])) {
+                // If current module is at the root of the workspace, there is no parent module
+                // to resolve . against. In VS there is always parent (project) but in VS Code
+                // there may be none. Treat `from . import test` as `import test`
```

Comment:
It doesn't behave like that, so we shouldn't treat it so. `from . import name` fails if you do not have a package (parent module): `ImportError: attempted relative import with no known parent package`

Reply:
It shouldn't work in VS either, since we don't treat the project as a package.

Reply:
You are right, doesn't work in VS either. OK, if it is not supposed to work, so be it.